cnf-tests/pod-utils/utils.go

```diff
@@ -0,0 +1,40 @@
+package pod_utils
+
+import (
+	"golang.org/x/sys/unix"
+	"io/ioutil"
+	"k8s.io/klog"
+	"k8s.io/kubernetes/pkg/kubelet/cm/cpuset"
+	"os/exec"
+	"strings"
+)
+
+func GetSelfCPUs() (*cpuset.CPUSet, error) {
+	cmd := exec.Command("/bin/sh", "-c", "grep Cpus_allowed_list /proc/self/status | cut -f2")
+	out, err := cmd.CombinedOutput()
+	if err != nil {
+		klog.Errorf("failed to run command, out: %s; err: %v", out, err)
+		return nil, err
+	}
+
+	cpus, err := cpuset.Parse(strings.Trim(string(out), "\n"))
+	if err != nil {
+		return nil, err
+	}
+
+	return &cpus, nil
+}
+
+func PrintNodeInformation() {
+	out, err := ioutil.ReadFile("/proc/cmdline")
+	if err != nil {
+		klog.Fatalf("failed to read file /proc/cmdline, err: %v", err)
```

Comment:
Is the difference in behaviour (fatal vs error) between this and GetSelfCPUs done purposely?
I believe Fatalf panics too.
I'd keep both consistent returning an error and exiting from main